observatory_platform/tests/test_dataset_api.py

```diff
@@ -61,7 +61,7 @@ def test_add_dataset_release(self):
             rows = bq_run_query(f"SELECT * FROM `{api.full_table_id}`")
             self.assertEqual(1, len(rows))
             actual = DatasetRelease.from_dict(dict(rows[0]))
-            self.assertEqual(expected, actual)
+            self.assertEqual(expected.to_dict(), actual.to_dict())
```

Comment:
The actual object should be compared (it is compared with the "__eq__" function), as the dict form is different.